python/py-keyczar/Portfile

```diff
@@ -25,13 +25,16 @@ python.versions     27
 platforms           darwin
 supported_archs     noarch
 
+patchfiles-append   patch-requires-pycryptodome.diff \
+                    patch-setup-pycryptodome.diff
```

Comment:
In retrospect, we could also accomplish this via `reinplace`, rather than two formal patches.

If anyone strongly prefers that approach, I'm happy to oblige!